dbt/adapters/bigquery/connections.py

```diff
@@ -787,7 +789,7 @@ def reopen_conn_on_error(error):
                 target=fn,
                 predicate=_ErrorCounter(self.get_job_retries(conn)).count_error,
                 sleep_generator=self._retry_generator(),
-                deadline=self.get_job_retry_deadline_seconds(conn),
+                timeout=self.get_job_retry_deadline_seconds(conn),
```

Comment:
it's just that the previous field is deprecated in the driver, it's the same behavior